src/components/icon/icons/svgs/bin/bin.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable max-len */
 export default {
   __html:
-    '<svg class="carbon-icon__svg carbon-icon__svg--bin" width="16px" height="16px" viewBox="0 0 16 16">'
+    '<svg class="carbon-icon__svg--bin" width="16px" height="16px" viewBox="0 0 16 16">'
```

Comment:
isn't the `fill` property modified with the removed here `carbon-icon__svg` class?

Reply:
These were removed by Iga, I think the functionality has been maintained though so it is okay to remove them

Reply:
I think that could have been removed by mistake. The `carbon-icon__svg` class was setting `fill: currentColor` and now I can't see it anywhere. It might affect all svg icons.

Reply:
Ok I will revert them